tests/test_groupby.py

```diff
@@ -246,15 +246,49 @@ def df2scalar(val: DataFrame) -> float:
         )
 
         # interpolate
-        check(assert_type(GB_DF.resample("ME").interpolate(), DataFrame), DataFrame)
-        check(
-            assert_type(GB_DF.resample("ME").interpolate(method="linear"), DataFrame),
-            DataFrame,
-        )
-        check(
-            assert_type(GB_DF.resample("ME").interpolate(inplace=True), None),
-            type(None),
-        )
+        try:
```

Comment:
should this be `if PD_LTE_22:`?

Reply:
Good point.  Not sure why I went that direction.  Fixed in https://github.com/pandas-dev/pandas-stubs/pull/920/commits/47aac27bde61fb6cd188b770c84f773eb529092b